index.js

```diff
@@ -8,7 +8,10 @@ module.exports = function () {
         "stan": "log",
         "sus": "warn",
         "cringe": "error",
-        "L": "Error"
+        "L": "Error",
+        "boss": "const",
+        "homie": "let",
+        "dude": "var"
     };
```

Comment:
These are great! Did this work when you ran it? I am getting an error.

Reply:
Sorry for the late reply, to be honest I did not test it, I had a look at it once again but I could not find a way to make VariableDeclarations work as `boss a = false` what I was able achieve was `boss("a", false)`. Let me know what you think?
Also could you run the tests I was unable to run on my local I tried `npx jest`.

Reply:
Ok, thanks! They would be funnier as regular declarations, but idk how to do it either... so this should work until we can figure it out!! Thanks so much again for contributing.

`npm run test` runs them, but the tests are kind of broken right now. I didn't have time to fix the issue so I'll make an issue for it.